gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -17,6 +17,7 @@
 package com.badlogic.gdx.graphics;
 
 import com.badlogic.gdx.utils.NumberUtils;
+import jdk.internal.joptsimple.internal.Strings;
```

Comment:
Is this public API?

Reply:
Nope.
[fix](https://github.com/libgdx/libgdx/pull/5670/commits/3ac9b4ce973d38e33b019b256b2fab6d321e7d8b)